src/semeio/workflows/ahm_analysis/ahmanalysis.py

```diff
@@ -292,8 +298,14 @@ def run(
 
 def _run_ministep(facade, prior_storage, target_storage, obs_group, data_parameters):
     # pylint: disable=too-many-arguments
+    rng = np.random.default_rng(seed=facade.config.random_seed)
```

Comment:
This makes me slightly nervous, what is the reasoning?

Reply:
Why does that make you nervous? It is just so it is reproducible. It was primarily for my snapshot test, but it is just using the seed the user has (optionally) provided.

Reply:
Created issue for saving random seed in ert: https://github.com/equinor/ert/issues/7499